app/views/application/tips.html.erb

```diff
@@ -0,0 +1,98 @@
+<main>
+  <%= javascript_include_tag 'index' %>
```

Comment:
Not too sure if the JS should be here (if anything it should be closer to the bottom) or it may already be included in one of the layouts

Reply:
yeah looks like we don't need it so I removed it.